dev/merge_arrow_pr.py

```diff
@@ -32,8 +32,8 @@
 # Configuration environment variables:
 #   - APACHE_JIRA_TOKEN: your Apache JIRA Personal Access Token
 #   - ARROW_GITHUB_API_TOKEN: a GitHub API token to use for API requests
-#   - PR_REMOTE_NAME: the name of the remote to the Apache git repo (set to
-#                     'apache' by default)
+#   - ORG_NAME: the name of the remote to the Apache git repo
```

Comment:
How about `ARROW_GITHUB_ORG` or `ARROW_GITHUB_OWNER` because `ARROW_GITHUB_API_TOKEN` has `ARROW_GITHUB_` prefix?

> the remote to the Apache git repo

Could you also update this description?